roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1024,6 +1041,7 @@ def generate_facts(self, local_facts, additive_facts_to_overwrite):
         facts = set_flannel_facts_if_unset(facts)
         facts = set_node_schedulability(facts)
         facts = set_master_selectors(facts)
+        facts = set_openshift_examples_facts_if_unset(facts)
```

Comment:
I don't think there is a need to set the default through a method, that is really only needed for values that don't have a simple default.  Instead, you should be able to set the default in get_defaults() instead.